### PR TITLE
182522755-dev-instructions

### PR DESCRIPTION
**Pull request by maciejkocylapc**
#### What's this PR do?
- simple seed update

#### How should this be manually tested?
- check the instructions.md file in keybase
- run `rails db:seed`

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182522755)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: script/add_current_epoch.rb [hunk 1]
```diff
@@ -43,8 +43,10 @@ def get_confirmed_block(http, uri, slot)
   else
     url = if network == 'mainnet'
       Rails.application.credentials.solana[:mainnet_urls][0]
+      # "https://api.mainnet-beta.solana.com"
     else
       Rails.application.credentials.solana[:testnet_urls][0]
+      # "https://api.testnet.solana.com"
```
Comment (kbiala) [inline script/add_current_epoch.rb:49]: What are these commets for?

File: db/seeds.rb [hunk 2]
```diff
@@ -56,4 +56,6 @@ def create_ping_things(number)
 
   create_users
   create_ping_things(240)
-end
\ No newline at end of file
+
+  load(Rails.root.join('script', 'add_current_epoch.rb') )
+end
```
Comment (kbiala) [inline db/seeds.rb:61]: Please also review other data that is created in seeds and can be removed, eg. there's no point in creating ping things in seeds. There are separate scripts for them. Also, code related to ping things in seed is not very good.

**Review by kbiala — CHANGES_REQUESTED** [review]

**Review by kbiala — COMMENTED** [review]
I'd also add a short mention about instruction in README in "Database population" section (maybe after "Database initialization")

File: db/seeds.rb (hunk 2 above)
Comment (radek-polcode) [reply db/seeds.rb:61]: It'd be nice to add puts statement to seeds that will tell the user where he should look for the scripts

**Pull request merged**